src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImpl.java

```diff
@@ -77,9 +77,9 @@ public NewUserResponse addNewUserToAnOrganisation(NewUserCreationRequest newUser
      */
     public ProfessionalUser findProfessionalUserByEmailAddress(String email) {
         ProfessionalUser user = professionalUserRepository.findByEmailAddress(email);
-        if (user == null) {
+        /*if (user == null) {

```

Comment:
Why is this commented out?

Reply:
Removed commented code. It is not required now and  handling logic in the controller.